### DISCUSSION TITLE
Update v6.6 - dev.3

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#206] [Pre-Release]
* add option to deactivate contacts provider
* set over-scroll color to the ripple color
* fix crash when result list is empty
* add tags menu as a gesture
* add tags menu preferences (tag icon toggle, tag list, tag order)
* add "remove from history" to shortcuts in result list
* improve "Uninstall" result list update

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.6-dev.3'>Update v6.6 - dev.3</a>.</em>

**Comment by ltguillaume** [comment]
1. I honestly thought that the tags list would just be the results list showing the tags as results...
2. ...and that there would be an action for the tags _list_ for the quick list, just as you can add a single tag to it.
3. If I set a gesture to open the new tags menu and then select a tag from it, triggering it from the widget mode just draws the results list (with a transparent background in my case) on top of the widgets. I guess there's no check if widgets are being hidden before a results list is drawn, since this also happens when I put a tag action into the quick list.
4. Could you add "Untagged" as possible tags list item?
5. I tried to rename tags via the tag manager, but it doesn't do anything.

**Reply by TBog** [reply]
1. Aha. Well the issue I was trying to solve was requesting the tags menu from KISS
2. I should be able to do that.
3. My mistake. Not enough testing. Glad I only released this as a dev/pre release
4. Sure.
5. I'll have a look. It was working when I released that feature.

**Reply by ltguillaume** [reply]
1. Aah ok, never really saw that in action, hence my confusion.
2. Cool!
3. Also tends to happen when I press back a few times rapidly in order to get out of an app, some kind of race condition I guess: I end up with the recents opened up on top of my widgets. I'm guessing the app is busy preparing the search mode while I already pressed back one more time, bringing it back to widget mode, but then the search mode opens on top of it as soon as it's done prepping.

**Reply by TBog** [reply]
Why do you need "Untagged" as an option?
You can already add a gesture or set it in the QuickList.

**Reply by ltguillaume** [reply]
> 
> 
> Why do you need "Untagged" as an option?
> You can already add a gesture or set it in the QuickList.

I'm using untagged for new apps I'm testing, so it's an actual group for me. Gestures are used up and adding a separate item for it in the quick list, next to tags list wouldn't really make sense imho and clutter the quick list unnecessarily.
